cmake/Findnl-route-3.cmake

```
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+#
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+find_library(NL3_ROUTE_LIBRARY "libnl-route-3.so")
```

I know this is copied from genl, but this really should be
```suggestion
find_library(NL_ROUTE_3_LIBRARY "libnl-route-3.so")
```

---

done.